.circleci/config.yml

```diff
@@ -57,6 +57,12 @@ jobs:
             echo "Cannot pass creds to forked PRs, so marking this step successful"
             circleci step halt
           fi
+    - &build
+      run:
+        name: Build
+        command: |
+          python3.8 -m venv venv/
+          venv/bin/pip install --upgrade -r requirements.txt
```

Comment:
The `&build` anchor is already defined earlier in this file, so we should be able to inject the contents here. I'm actually confused about the `<<: *build` line earlier in the `integration` job configuration, as that looks to be injecting content from the anchor, but at the wrong level here. I may have missed that in an earlier review. I can't remember exactly what the right yaml syntax should be here.

Reply:
Figured it out.